collector/main.go

```diff
@@ -3,15 +3,24 @@ package main
 import (
 	"github.com/Kindling-project/kindling/collector/application"
 	"log"
+	"fmt"
```

Comment:
The compiler would complain about an error if an imported package is not used. I don't find the usage of `fmt` in this file so I think it is safe to remove this line.

Reply:
Oh, sorry! I slightly modified it before submitting but did not test it. I was negligent. Speaking of this, I would like to ask whether any changes to the program should be tested through the whole process of compiling, mirroring, uploading and deploying to the cluster. For example, this function just does small changes in the `main()`, but it takes a long time to verify it through the whole process.